287_find_duplicate_number.py

```diff
@@ -0,0 +1,20 @@
+# time complexity O(logn)
+# space complexity O(1)
```

Comment:
Space complexity is wrong.
Investigate space complexity of Python's sort method.